objects.js

```diff
@@ -0,0 +1,69 @@
+// Objects 1. BetweenDates
+
+function durationBetweenDates(startDate = new Date(), endDate = new Date(), dimension = 'seconds') {
```

Comment:
The wrong type of default values of dates. We need to have strings here not Dates.